### ISSUE TITLE
how to analyze the memory heap by the heapdump file？

### ISSUE THREAD

**Issue by chelsh** [bnoordhuis/node-heapdump#58]
![image](https://cloud.githubusercontent.com/assets/10752360/8541187/d90e3e56-24be-11e5-8aa0-caa9969b8dc9.png)

**Comment by bnoordhuis** [comment]
Your question is too open-ended.  Have you followed the Chrome DevTools tutorial?  If you did but things still aren't clear, can you rephrase your question?  Closed yes/no-style questions are preferable.  :-)

**Comment by geekyme** [comment]
@chelsh https://strongloop.com/strongblog/how-to-heap-snapshots/ everything you need here.

**Comment by bnoordhuis** [comment]
Good answer, thanks.  I'll close the issue.
